docs/docs.c

```diff
@@ -40,7 +40,7 @@
 ///@li @ref mip_packet_c
 ///@li @ref mip_field_c
 ///@li @ref mip_parser_c
-///@li @ref mip::C::mip_cmd_result
+///@li @ref mip::C::mip_cmd_result "mip_cmd_result [C]"
```

Comment:
not sure what this does?

Reply:
This overrides the text of the link so it doesn't have the `mip::C::` prefix.